pygments/lexers/blueprint.py

```diff
@@ -0,0 +1,240 @@
+"""
+    pygments.lexers.blueprint
+    ~~~~~~~~~~~~~~~~~~~~~~~~~
+
+    Lexer for the Blueprint UI markup language.
+
+    :copyright: Copyright 2006-2023 by the Pygments team, see AUTHORS.
+    :license: BSD, see LICENSE for details.
+"""
+
+import re
+
+from pygments.lexer import RegexLexer, include, bygroups, words
+from pygments.token import (
+    Comment,
+    Operator,
+    Keyword,
+    Name,
+    String,
+    Number,
+    Punctuation,
+    Whitespace,
+)
+
+__all__ = ["BlueprintLexer"]
+
+
+class BlueprintLexer(RegexLexer):
+    """
+    For Blueprint UI markup. See https://gitlab.gnome.org/jwestman/blueprint-compiler.
+
+    .. versionadded:: 2.16
+    """
+
+    name = "Blueprint"
+    aliases = ["blueprint"]
+    filenames = ["*.blp"]
+    mimetypes = ["text/x-blueprint"]
+
```

Comment:
Please use `url = "https://gitlab.gnome.org/jwestman/blueprint-compiler"` instead of putting it into the doc string, this will allow linking to the website in various places.